plugin/trino-mariadb/src/test/java/io/trino/plugin/mariadb/TestingMariaDbServer.java

```diff
@@ -48,11 +48,26 @@ public TestingMariaDbServer(String tag)
         execute(format("GRANT ALL PRIVILEGES ON *.* TO '%s'", container.getUsername()), "root", container.getPassword());
     }
 
+    public TestingMariaDbServer(String tag, String histogramType)
```

Comment:
Please add a new argument to the existing constructor.

Reply:
Done